### DISCUSSION TITLE
v1.7 - More Comamnds

### DISCUSSION THREAD

**Discussion by ShimmyMySherbet** [ShimmyMySherbet/ShimmysAdminTools#15] [Announcements]
* Added /LaunchBushes {Player}
* Added /Firework [Player]
* Added /DestroyResources [Range]
* Added /ReviveResources [Range]
* Added /Boom
* Added /KickAll [Reason]
* Added /AlertZombies [Player]
* Added /SimFire [Player]
* Added /Trail [Trail]

<hr /><em>This discussion was created from the release <a href='https://github.com/ShimmyMySherbet/ShimmysAdminTools/releases/tag/v1.7'>More Comamnds</a>.</em>

**Comment by TheHitmanLove** [comment]
Здравствуйте, а куда устанавливать данные файлы?
